### DISCUSSION TITLE
Add ability to mute alerting when deployments are scaled down by autoscaling

### DISCUSSION THREAD

**Discussion by imdmahajankanika** [okd-project/okd#1589] [General]
**Describe the bug**

We want to setup auto-scaling in our OKD cluster for the better utilisation of resources. But is there a way to silence the alerts automatically when the deployments are scaled down by autoscaler?

**Version**

**How reproducible**

**Log bundle**

**Comment by vrutkovs** [comment]
https://docs.okd.io/latest/monitoring/managing-alerts.html#silencing-alerts_managing-alerts

**Reply by imdmahajankanika** [reply]
@vrutkovs  Following the above document, we have to specify the complete date for starting and ending the silence. 
But, for example, if we want our application to scale down automatically during the weekends and we want to mute the alerts "ApplicationNotReachable" during that period, how can we achieve that?
